spec/requests/stories_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+
+RSpec.describe 'Stories API', type: :request do
+
+  let!(:stories) { create_list(:story, 10) }
+  let(:story_id) { stories.first.id }
+
+  # Test suite for GET /stories
+  describe 'GET /stories' do
+    # make HTTP get request before each example
+    before { get '/stories' }
+
+    it 'returns stories' do
+      # Note `json` is a custom helper to parse JSON responses
+      expect(json).not_to be_empty
+      expect(json.size).to eq(10)
+    end
+
+    it 'returns status code 200' do
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  # Test suite for GET /stories/:id
+  describe 'GET /stories/:id' do
+    before { get "/stories/#{story_id}" }
+
+    context 'when the record exists' do
+      it 'returns the story' do
+        expect(json).not_to be_empty
+        expect(json['id']).to eq(story_id)
+      end
+
+      it 'returns status code 200' do
+        expect(response).to have_http_status(200)
+      end
+    end
+
+    context 'when the record does not exist' do
+      let(:story_id) { 100 }
+
+      it 'returns status code 404' do
+        expect(response).to have_http_status(404)
+      end
+
+      it 'returns a not found message' do
+        expect(response.body).to match(/Couldn't find Story/)
+      end
+    end
+  end
+
+  # Test suite for POST /stories
+  describe 'POST /stories' do
+    # valid payload
+    let(:valid_attributes) { { title: 'Learn Elm' } }
```

Comment:
This should prob include all attributes (aka add description for now), and then validate below that all are saved accordingly